src/api/migraphx.py

```diff
@@ -178,14 +178,67 @@ def shapes(h):
              returns='const migraphx::shape&')
 
 
+@auto_handle()
+def instruction_ref(h):
```

Comment:
I think maybe we should call this `instruction` just to be consistent with `module`.